E2ETests/Runner/AppConfiguration.swift

```diff
@@ -0,0 +1,67 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+import Foundation
+import DatadogInternal
+import DatadogCore
+
+/// Test info reads configuration from `Info.plist`.
+///
+/// The expected format is as follow:
+///
+///     <dict>
+///         <key>DatadogConfiguration</key>
+///         <dict>
+///             <key>ClientToken</key>
+///             <string>$(CLIENT_TOKEN)</string>
+///             <key>ApplicationID</key>
+///             <string>$(RUM_APPLICATION_ID)</string>
+///         </dict>
+///     </dict>
+struct TestInfo: Decodable {
+    let clientToken: String
+    let applicationID: String
+    let site: DatadogSite?
+    let env: String?
```

Comment:
question/ What about `site` and `env` in expected format? It is quite unclear to me what is expected vs not to be found in `Info.plist`.

Reply:
Indeed, it's not clear. I will make them non-optional with default values in xcconfig